code/game/mob/living/carbon/human/unarmed_attack.dm

```diff
@@ -127,8 +127,6 @@ var/global/list/sparring_attack_cache = list()
 /datum/unarmed_attack/punch
 	attack_verb = list("punched")
 	attack_noun = list("fist")
-	eye_attack_text = "fingers"
```

Comment:
Why???

Reply:
Because the only instance we use this is for humans, it doesn't grab the variable from the datum due to how it works.